src/servers/CMakeLists.txt

```diff
@@ -241,6 +241,7 @@ target_link_libraries(
   PRIVATE triton-common-error
   PRIVATE triton-core-serverapi
   PRIVATE tritonserver
+  PUBLIC pthread dl
```

Comment:
One library per line, and you should be able to use PRIVATE, so

PRIVATE pthread
PRIVATE dl

Reply:
Got it, will update

Reply:
Updated